### PR TITLE
LPS-156102 Poshi Automation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-156102

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/fragment/FragmentsUpgrade.testcase [hunk 1]
```diff
@@ -16,13 +16,37 @@ definition {
...
+
+			FragmentsEditor.typeHtmlEditor(content = "${htmlSnippet}");
+
+			FragmentsEditor.waitForAutoSave();
+
+			Button.clickPublish();
+
+			AssertConsoleTextNotPresent(value1 = "javax.portlet.PortletException:");
+
+			Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/fragment/FragmentsUpgrade.testcase:47]:
```suggestion
			FragmentsEditor.publish();

			AssertConsoleTextNotPresent(value1 = "javax.portlet.PortletException:");
```

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Would you mind double checking if this issue happens on upgrade from 7.1.10.5?
We already have case to test edit fragment upgrade from 7.2.10.1
**PageEditorUpgrade.ConfigurePageEditorAfterUpgrade72101**

**Comment by ozysouza** [comment]
> Hi @ozysouza, Would you mind double checking if this issue happens on upgrade from 7.1.10.5? We already have case to test edit fragment upgrade from 7.2.10.1 **PageEditorUpgrade.ConfigurePageEditorAfterUpgrade72101**

Hi @Tim-Cao, I also can reproduce the issue on the test you sent. The issue I could find is the same when I was writing these steps to validate the bug.

The macro FragmentsEditor.publish(); doesn't work as expected, the Error is thrown on UI when trying to save the fragment but the validation Alert.viewSuccessMessage(); doesn't work, even thoug it expects the success message that doens't appear, the text keeps running untill finish with success. Because of it added steps separately 

                    Button.clickPublish(); - Publish

			AssertConsoleTextNotPresent(value1 = "javax.portlet.PortletException:"); - Break the test if the error is thrown on console

			Alert.viewSuccessMessageText(successMessage = "Your request completed successfully."); - If no error is thorn also break the test if the success message is not shown
			
The main issue is that if the Sucesse message is not shown the function VerifyElementPresent(locator1 = "Message#SUCCESS_DISMISSIBLE"); will not break the test.

We would follow three ways;

1st - Keep the changes I made
2nd - Add these three steps  below inside Upgrade.configurePageEditor(); which will cover the issue

		task ("Assert could edit fragment upgrading from 7.2") {
			FragmentsAdminNavigator.openFragmentsAdmin(siteURLKey = "test-site-name");

			FragmentsAdminNavigator.gotoEntry(entry = "Fragment Name");

			FragmentsEditor.typeFileContent(fileName = "fragment_title.html");

			FragmentsEditor.typeFileContent(fileName = "fragment_title.css");

			FragmentsEditor.assertFragmentPreview(
				fragmentLocator = "//div[@class='fragment-title']/h1[@class='title']",
				text = "LOREM IPSUM DOLOR SIT AMET");

			Button.clickPublish();

			AssertConsoleTextNotPresent(value1 = "javax.portlet.PortletException:");

			Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
		}


3rd - Try to understand why the test is not breaking on the function VerifyElementPresent (I've already tried this one but unsuccessfully until now)

Waiting for you reply.

**Comment by Tim-Cao** [comment]
I could say the Option 2. VerifyElementPresent won't break the test is expected, it will shown a warning highlight on test report.

**Comment by ozysouza** [comment]
> Thanks for your testing @ozysouza I could say the Option 2. VerifyElementPresent won't break the test is expected, it will be shown as a warning highlight on test report.

@Tim-Cao Thanks for the explanation about the function. I've updated th PR, as it's just a change that you have already agreed I will forward the PR to earn some time.

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**